statchart/src/utils/get-color.ts

```diff
@@ -27,7 +27,12 @@ export function getStatChartColor(
   // Determine the default color from thresholds or theme
   const defaultColor = thresholds?.defaultColor ?? chartsTheme.thresholds.defaultColor;
 
-  if (!value || (!thresholds?.steps && !mappings)) {
+  if (
+    value === undefined ||
+    value === null ||
+    (typeof value === 'string' && !value) ||
```

Comment:
I guess this can be simplified to 
```suggestion
    value === '' ||
```